test/integration/fixtures/esm/test-that-uses-dir-cjs-require.fixture.js/index.js

```diff
@@ -0,0 +1,4 @@
+// See https://github.com/mochajs/mocha/issues/4665 for an explanation of this test
```

Comment:
Why is this fixture a directory? This is the test file, not the module to be required via `index.js`.

Reply:
I wanted to check both the loading of a test file that is a directory that has `index.js`, _and_ a loading using `--require` of a similar directory. The bug is in the `--require`, but I wanted to have a test for both just to ensure that it works whatever the changes we do in how we import the test files.

Reply:
https://github.com/mochajs/mocha/pull/4666